packages/@remirror/core/src/manager/remirror-manager.ts

```diff
@@ -347,19 +339,19 @@ export class RemirrorManager<Combined extends AnyCombinedUnion> {
       const destroyHandler = extension.onDestroy;
 
       if (createHandler) {
-        this.#handlers.create.push(createHandler);
+        this.#handlers.create.push(createHandler.bind(extension));
```

Comment:
@ifiokjr You may need to check this modification.

Reply:
This looks good to me. Are there any potential pitfalls?